docs/src/samples/cluster-example-logical-dest.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cluster-example-dest
+spec:
+  instances: 1
+
+  storage:
+    size: 1Gi
+
+  bootstrap:
+    initdb:
+      import:
+        type: microservice
+        schemaOnly: true
+        databases:
+          - app
+        source:
+          externalCluster: cluster-example
+
+  externalClusters:
+  - name: cluster-example
+    connectionParameters:
+      host: cluster-example-rw.default.svc
+      # We're using the superuser to allow the publication to be
+      # created directly when connected to the target server.
+      # See cluster-example-logical-source.yaml for more information
+      # about this.
+      user: postgres
+      dbname: app
+    password:
+      name: cluster-example-superuser
+      key: password
+
+# To create the publication while connected to the target side, use:
```

Comment:
"while connected" ... not sure what this means here

Reply:
The commands are abstracting us from where they get executed.

Reply:
I removed the cnpg invocation from the examples. They're too far away from the code to be really kept updated.